### ISSUE TITLE
Breaks bash completion of SCP (and probably others)

### ISSUE THREAD

**Issue by psxpaul** [gradle/gradle-completion#29]
Without gradle-completion sourced in my .bashrc, I can do tab completion with scp:
scp remote_server:.bash_<press tab key>
...
scp remote_server:.bash_history

With gradle-completion sourced, I get:
scp remote_server:.bash_<press tab key>
...
scp .bash_history

I believe it is because COMP_WORDBREAKS is modified at the start of the script, which could effect any bash_completion script that uses the ':' character. I commented out this line in gradle-completion.bash, and scp completion began working again.

**Comment by yschimke** [comment]
This breaks completion for my programs.  Changing COMP_WORDBREAKS must be avoided.

n.b. you can achieve the same thing using 

```
  _get_comp_words_by_ref -n : cur
...
  __ltrim_colon_completions "$cur"
```

**Comment by meonlol** [comment]
@psxpaul @yschimke: Since this doesn't seem to get addressed, you could use my gradle-completion script if you like: https://github.com/meonlol/gradle-tab-completion
Any feedback is appreciated.

**Comment by eriwen** [comment]
This is fixed by e74489be30cb5f7e552a840e47523b5ea99b796f
